src/jsonschema.rs

```diff
@@ -69,6 +70,7 @@ struct Array {
 #[serde(rename_all = "kebab-case")]
 enum Format {
     DateTime,
+    Bytes,
```

Comment:
I would add a comment, something to the effect of "Custom format value for casting strings into byte-strings".